test/hotspot/gtest/runtime/test_os_windows.cpp

```diff
@@ -722,6 +722,105 @@ TEST_VM(os_windows, processor_count) {
   }
 }
 
+TEST_VM(os_windows, large_page_init_multiple_sizes) {
+  // Enable large pages and set conditions for the test
+  UseLargePages = true;
+  bool isSupportedWindowsVersion = os::win32::is_windows_11_or_greater() ||
+    os::win32::is_windows_server_2022_or_greater();
+  EnableAllLargePageSizesForWindows = true; // This flag enables support for all large page sizes
+
+  // Determine the minimum page size
+  const size_t min_size = GetLargePageMinimum();
+
+  // Set LargePageSizeInBytes to 4 times the minimum page size
+  LargePageSizeInBytes = 4 * min_size; // Set a value for multiple page sizes
+
+  // Initialize large page settings
+  os::large_page_init();
+
+  // Verify that large pages are enabled
+  if (isSupportedWindowsVersion) {
+    EXPECT_TRUE(UseLargePages) << "UseLargePages should be true after initialization for LargePageSizeInBytes = 4 * min_size";
+  }
+
+  // Verify that _large_page_size is greater than the default page size
+  const size_t default_page_size = os::vm_page_size();
+  size_t _large_page_size = os::large_page_init_decide_size();
```

Comment:
nit: a leading underscore is only for class data member names, not local variables.